src/mappings/modules/core/vat.ts

```diff
@@ -321,18 +322,23 @@ export function handleSuck(event: LogNote): void {
 // Modify the debt multiplier, creating/destroying corresponding debt
 export function handleFold(event: LogNote): void {
   let ilk = event.params.arg1.toString()
-  let rate = units.fromRay(bytes.toSignedInt(event.params.arg3))
+  let u = bytes.toAddress(event.params.arg2)
```

Comment:
nitpick: name variable. Replacing it by kind of user_bytes or something like this?